apps/core/scenes/bookingThankYou/ThankYou.js

```diff
@@ -0,0 +1,161 @@
+// @flow
+
+import * as React from 'react';
+import {
+  withNavigation,
+  type Navigation,
+  Routes,
+} from '@kiwicom/margarita-navigation';
+import {
+  Illustration,
+  ContentContainer,
+  Separator,
+} from '@kiwicom/margarita-components';
+import {
+  StyleSheet,
+  Text,
+  Button,
+  RadioButton,
+} from '@kiwicom/universal-components';
+import { defaultTokens } from '@kiwicom/orbit-design-tokens';
+
+import InformationSection from '../../components/InformationSection/InformationSection';
+
+type Props = {|
+  +navigation: Navigation,
+|};
+
+type travellingForType = 'business' | 'fun';
+
+type State = {|
+  +isTravellingFor: travellingForType,
+|};
+
+class ThankYou extends React.Component<Props, State> {
+  state = {
+    isTravellingFor: 'business',
+  };
+
+  handleConfirmBooking = () => {
+    this.props.navigation.navigate(Routes.RESULTS);
+  };
+
+  handleTravellingForChange = (change: travellingForType) => {
+    this.setState({ isTravellingFor: change });
+  };
+
+  setTravellingForFun = () => {
+    this.handleTravellingForChange('fun');
+  };
+
+  setTravellingForBusiness = () => {
+    this.handleTravellingForChange('business');
+  };
+
+  render() {
+    return (
+      <ContentContainer>
+        <Illustration name="Feedback" style={styles.illustration} />
+        <Text
+          style={styles.marginVerticalSmall}
+          align="center"
+          size="large"
+          weight="bold"
+          type="primary"
+        >
+          Thank you for booking at Kiwi.com
```

Comment:
`for booking [what] at Kiwi.com`

Maybe "your flight"?

Reply:
Done!